client/my-sites/domains/domain-management/domain-mapping-instructions/index.jsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { isSubdomain } from 'calypso/lib/domains';
+import {
+	MAP_DOMAIN_CHANGE_NAME_SERVERS,
+	MAP_EXISTING_DOMAIN_UPDATE_A_RECORDS,
+	MAP_SUBDOMAIN,
+} from 'calypso/lib/url/support';
+import { WPCOM_DEFAULT_NAMESERVERS } from 'calypso/state/domains/nameservers/constants';
+import FoldableFAQ from 'calypso/components/foldable-faq';
+import { Notice } from 'calypso/components/notice';
+import ExternalLink from 'calypso/components/external-link';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class DomainMappingInstructions extends React.Component {
+	static propTypes = {
+		domain: PropTypes.object,
+		isAtomic: PropTypes.bool,
+		isLoaded: PropTypes.bool,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		isAtomic: false,
+		isLoaded: false,
+	};
+
+	render() {
```

Comment:
By convention, `render` is usually the last method in the component. The way I've always approached it was go from most internal to most "public" ones. So, lifecycle methods like `willReceiveProps`, etc. first and then smaller `render*` methods. And then finally the main `render` method.